build/crd-samples/router/create-rule-rest-eventbus.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rules.kubeedge.io/v1
+kind: Rule
+metadata:
+  name: my-rule
+  labels:
+    description: test
+spec:
+  source: "my-rest"
+  sourceResource: {"path":"/a"}
```

Comment:
Please use more meaningful path, there are many `/a` and `/b` which would confuse new users.

Reply:
this is only an example, user can define by self to replace /a...